DOCKER.md

```
@@ -0,0 +1,206 @@
+## 🛠️ Local Development Setup
+
+```bash
+# Clone and setup everything
+git clone https://github.com/Quegenx/vercel-mcp-server.git
+cd vercel-mcp-server
+./scripts/setup.sh
+```
+
+## 🔧 Manual Setup
+
+### 1. Prerequisites
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)